pkgs/by-name/go/goutline/package.nix

```diff
@@ -0,0 +1,35 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  go,
+  lib,
+}:
+
+buildGoModule rec {
+  pname = "goutline";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "1pkg";
+    repo = "goutline";
+    rev = "v${version}";
+    hash = "sha256-YZM+pQobuFGlxX+t1rgiM8JMc4n1AWveQr0W3LE9yPc=";
+    postFetch = ''
+      export GOCACHE=$TMPDIR/go-build
+      export GOPATH=$TMPDIR/go
+      export GOMODCACHE=$TMPDIR/go/pkg/mod
+      cd $out
+      ${lib.getExe go}/bin/go mod init github.com/1pkg/goutline
```

Comment:
`lib.getExe go` will be the full path to the executable so you’ll want to remove the `/bin/go`.

Reply:
Fixed, thank you!